### PR TITLE
cleanup some code in staging/src/k8s.io

### PR DESCRIPTION
**Pull request by wgliang**
**What this PR does / why we need it**:
/kind cleanup

cleanup some code in staging/src/k8s.io

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by wgliang** [comment]
/cc @lavalamp @smarterclayton @deads2k
PTAL,thanks.

**Comment by wgliang** [comment]
/cc @lavalamp @smarterclayton @deads2k
PTAL,thanks.

File: staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protobuf/generator.go [hunk 1]
```diff
@@ -371,7 +371,7 @@ func (b bodyGen) doStruct(sw *generator.SnippetWriter) error {
 `, b.t)
 
 	if len(options) > 0 {
-		sort.Sort(sort.StringSlice(options))
+		sort.Strings(options)
```
Comment (deads2k) [inline staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protobuf/generator.go]: Seems silly, but I don't actually know that these things are equivalent.   Mind linking me to a golang playground?

Comment (wgliang) [reply staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protobuf/generator.go]: https://play.golang.org/p/_xl4AUHU_y9

**Pull request closed without merging**